nextflow_schema.json

```diff
@@ -0,0 +1,673 @@
+{
```

Comment:
Please update with all parameters from `parameters.md` - should be very easy to do so using `nf-core schema build` on the pipeline - even nice webinterface available to do this :-) 

=> then you can delete the `parameters.md` file :-)

Reply:
Done. 
We put a lot of work into setting that parameters page, so would like to keep it. 
We agree that the tool is very useful, so have added text explaining the tool and a hyperlink directing the user to the page - we've put this at the top of the parameters page.